components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -12,23 +12,23 @@ toc = true
 
 ## Features in IAM v2
 
-IAM v2 policies allow multiple permissions, separating out policy membership from policy definition for fine-grained control, and includes roles for role-based access control.
-Additionally, IAM v2 allows policy members to be managed directly from the Automate UI.
+IAM v2 policies allow multiple permissions, distinguish policy membership from policy definition for fine-grained control, and include roles for role-based access control.
+Additionally, IAM v2 allows policy members to be managed directly from Chef Automate in the browser.
 Perhaps most significantly, IAM v2 supports projects, which allow for filtering and segregation of your data amongst your user base.
 
-At the heart of Chef Automate's IAM system is the *policy*.
+The *policy* exists at the heart of Chef Automate's IAM system.
 A policy defines permissions for who may perform what action on which resource scoped by project.
 The "who" may be a user, a team, or an API token.
 
-The following diagram shows the policy structure. We will detail the specifics in the next few sections.
+The following diagram shows the new policy structure. We detail the specifics in the next few sections.
 
 ![](/images/docs/iam-v2-diagram.png)
 
 ## Policy Definition
 
 IAM v2 uses multi-statement policies that support complex permissions in a single policy.
 Each statement specifies a single permission.
-The net effect (ALLOW or DENY) of a policy is determined by evaluating the effect of each statement and combining them. If there is at least one statement allowing access, and none denying it, then the policy allows access. Otherwise, access is denied.
+Evaluating the effect of each statement and combining them determines the net effect (ALLOW or DENY) of a policy. If at least one statement allows access, and none denies it, then the policy allows access. Otherwise, access denied.
```

Comment:
I can't tell here if the `them` in `combining them` is the statement or the effect. (I know grammatically it's the effect, but for some reason it is still sticky for me.) Maybe something like "The effect in each statement is evaluated in conjunction with the effects in other statements"? I dunno. Words are hard.